SPDX-FileCopyrightText: 2022 fosslinux <fosslinux@aussies.space>

SPDX-License-Identifier: PSF-2.0

openssl is too new for this version of Python. Tell Python build system
we don't have openssl.

--- Python-3.1.5/setup.py	2022-12-19 10:51:49.749157041 +1100
+++ Python-3.1.5/setup.py	2022-12-19 10:52:37.223748681 +1100
@@ -712,7 +712,7 @@
 
         #print('openssl_ver = 0x%08x' % openssl_ver)
 
-        if ssl_incs is not None and ssl_libs is not None:
+        if False:
             if openssl_ver >= 0x00907000:
                 # The _hashlib module wraps optimized implementations
                 # of hash functions from the OpenSSL library.
@@ -727,12 +727,12 @@
         else:
             missing.append('_hashlib')
 
-        if openssl_ver < 0x00908000:
+        if True:
             # OpenSSL doesn't do these until 0.9.8 so we'll bring our own hash
             exts.append( Extension('_sha256', ['sha256module.c']) )
             exts.append( Extension('_sha512', ['sha512module.c']) )
 
-        if openssl_ver < 0x00907000:
+        if True:
             # no openssl at all, use our own md5 and sha1
             exts.append( Extension('_md5', ['md5module.c']) )
             exts.append( Extension('_sha1', ['sha1module.c']) )
